packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -195,21 +195,32 @@ export interface IContainerRuntimeBase extends IEventProvider<IContainerRuntimeB
 	 * already attached DDS (or non-attached DDS that will eventually get attached to storage) will result in this
 	 * store being attached to storage.
 	 * @param pkg - Package name of the data store factory
-	 * @param groupId - group to which this data stores belongs to. This is also known at service side and can be used to
-	 * fetch snapshot contents like snapshot tree, blobs using this id from the storage.
-	 */
-	createDataStore(pkg: string | string[], groupId?: string): Promise<IDataStore>;
+	 * @param dataStoreGroupIdForSnapshotFetch - This represents the group of the datastore within a container or its snapshot.
+	 * When not specified the datastore will belong to a `default` group. When a container is loaded initially, only datastores
```

Comment:
I'd suggest not to put any text here at all, and only have a reference to readme.md
Alternatively, have such block not repeated - have in one place, and just reuse documentation (use @inheritDoc tag)

Reply:
I would like to keep doc on these apis as these are the base apis and everyone does does not go through a Readme. 
I will link to this place when using these somewhere else.

Reply:
I agree with Vlad on this. 'create' API annotation doesn't need to speak to how datastores are going to be 'fetched' using GroupId. That's more of a detail for the fetch logic not create logic. Another challenge with this annotation would be to always keep it up to date. When your readMe changes to reflect a new way of fetching, there's a high chance, the future engineer (or you) will forget to update this copy of comments.

Reply:
Reduced the comments here.